### PR TITLE
Add formula for hive 0.10.0

### PR DESCRIPTION
**Pull request by onlynone**
* There's a backwards incompatible bug between hive 0.13.0 and earlier
   versions that make having this version around nice
 * HIVE-6050

### CODE REVIEW

File: hive010.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+require 'formula'
```
Comment (DomT4) [inline hive010.rb]: :scissors: this line

File: hive010.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+class Hive < Formula
+  homepage 'http://hive.apache.org'
+  url 'http://archive.apache.org/dist/hive/hive-0.10.0/hive-0.10.0-bin.tar.gz'
```
Comment (DomT4) [inline hive010.rb]: Double quotes everywhere, Thanks!

File: hive010.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
+require 'formula'
+
+class Hive < Formula
+  homepage 'http://hive.apache.org'
```
Comment (DomT4) [inline hive010.rb]: Both this and the ` url ` should be ` https:// ` links.

File: hive011.rb [hunk 4]
```diff
@@ -0,0 +1,26 @@
+require 'formula'
```
Comment (DomT4) [inline hive011.rb]: Same comments on this formula as the one above.

**Comment by DomT4** [comment]
If this is still useful I've left some audit comments above that need addressing. If it isn't useful at this point, I'll close this at some point in the next week.

**Comment by onlynone** [comment]
Just wondering why this was closed. Was there any followup to the formulas?

**Comment by DomT4** [comment]
@onlynone Apologies - The lack of comment in reply and Git's way of displaying when something was first edited as opposed to when it was most recently edited threw me off here completely. I've reopened the PR and will ask the bot to take another look.

**Comment by DomT4** [comment]
@BrewTestBot test this please

File: hive010.rb [hunk 5]
```diff
@@ -0,0 +1,24 @@
+class Hive < Formula
```
Comment (DomT4) [inline hive010.rb]: Both these names need changing. ` class Hive010 ` and ` class Hive011 ` respectively.

**Comment by onlynone** [comment]
Should I summon @BrewTestBot , or should new commits trigger it automatically?

**Comment by DomT4** [comment]
New commits trigger it automatically. It hunts for SHA changes, which a new commit causes.

**Comment by DomT4** [comment]
Couple of minor comments from the test-bot and this should be okay:

```
* Stable: SHA1 checksums are deprecated, please use SHA256
 * Use `depends_on :java` to set JAVA_HOME
```

**Comment by DomT4** [comment]
Ping.

**Comment by DomT4** [comment]
Merged! Thanks for your contribution to Homebrew, we appreciate it! :smiley_cat:

**Pull request closed without merging**